src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkSWASectionSubmitted.tsx

```diff
@@ -1,13 +1,109 @@
-import { Flex, Text, WinningBidIcon } from "@artsy/palette"
+import {
+  Button,
+  Clickable,
+  Flex,
+  ModalDialog,
+  Text,
+  WinningBidIcon,
+} from "@artsy/palette"
+import { toTitleCase } from "@artsy/to-title-case"
+
+import { useState } from "react"
 import { RouterLink } from "System/Router/RouterLink"
+import { useFeatureFlag } from "System/useFeatureFlag"
+import { Media } from "Utils/Responsive"
+
+// TODO:- We are using displayText for Statuses for now. Consider changing the logic when proper statuses are made available on Metaphysics.
+// See https://artsyproduct.atlassian.net/browse/SWA-217
+// same as on Eigen
+const STATUSES: { [key: string]: { color: string; text: string } } = {
+  "submission in progress": { color: "yellow150", text: "In Progress" },
+  "submission evaluated": { color: "orange150", text: "Evaluation Complete" },
+  sold: { color: "black100", text: "Artwork Sold" },
+}
+
+interface Props {
+  submissionStatus: string
+}
+
+export const MyCollectionArtworkSWASectionSubmitted: React.FC<Props> = ({
+  submissionStatus,
+}) => {
+  const [
+    isSubmissionStatusModalOpen,
+    setIsSubmissionStatusModalOpen,
+  ] = useState(false)
+
+  const isMyCollectionPhase8Enabled = useFeatureFlag(
+    "my-collection-web-phase-8-submission-status"
+  )
 
-export const MyCollectionArtworkSWASectionSubmitted = () => {
   const article =
     "https://support.artsy.net/hc/en-us/sections/360008311913-Sell-with-Artsy"
 
+  const approvedDisplayText = STATUSES[submissionStatus!.toLowerCase()]?.text
```

Comment:
**thought:** is this text on for approved submissions? if not I would suggest maybe renaming this 👀

Reply:
maybe `displayText` or so